Applications/Tizen_native/CustomShortcut/README.md

```diff
@@ -16,14 +16,31 @@ You can use `tizen studio gui` to build and run.
 
 If you want to do in CLI, you first need to [convert the project to CLI](https://developer.tizen.org/ko/development/tizen-studio/native-tools/cli/converting-projects-cli) and do as follows.
 
-for the time being, you have to install and prepare nntrainer, capi-nntrainer to run this program
 
 ### Prerequisite
 
 - [NNTrainer Prerequisite](https://github.com/nnstreamer/nntrainer#prerequisites)
 - tizen studio 3.7
 - wearable 6.0 rootstrap (built after Aug 25 2020)
 - appropriate wearable tizen device with tizen 6.0 installed (built after Aug 25 2020)
+- gstreamer plugin png image decoder (https://gstreamer.freedesktop.org/documentation/png/pngdec.html?gi-language=c)
+
+### Install `pngdec`
```

Comment:
Instead of guiding to install pngdec, How about using just bmp format for input image. Then we don't need to install pngdec.

Reply:
It can't be easily done for now as [converting Cairo to png](https://www.cairographics.org/manual/cairo-PNG-Support.html) can be done in a line of code while there is no api to convert other formats. 
I need to deal with pixels, so it will take some time. I'll change to another format with less priority :)

Reply:
@zhoonit Can you once refer https://docs.tizen.org/application/native/guides/multimedia/image-edit/ for different image formats?

Reply:
That's neat. thanks.